htdocs/main.php

```diff
@@ -63,6 +63,7 @@ function tplFromRequest($param)
 $tpl_data['currentyear'] = date('Y');
 $tpl_data['test_name']   = $TestName;
 $tpl_data['subtest']     = $subtest;
+$tpl_data['version']     = file_get_contents('../VERSION');
```

Comment:
This should probably explicitly be relative to the dir of main.php, not the cwd: `__DIR__ . '../VERSION'`